console/console-init/ui/src/modules/iot-device-detail/components/DeviceDetailHeader/DeviceDetailHeader.tsx

```diff
@@ -236,7 +235,7 @@ const DeviceDetailHeader: React.FunctionComponent<IDeviceDetailHeaderProps> = ({
         <DeviceDetailLayout />
         <SplitItem isFilled />
         <SplitItem className={css(styles.kebab_toggle_margin)}>
-          <Switch
+          <SwitchWithToggle
             id="switch-device-header-enable-btn"
```

Comment:
Try to unify ids so it in most of the console is used 'button' change this ID to  switch-device-header-enable-button